src/Validation/Validation.php

```diff
@@ -1424,6 +1424,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a scalar.
+     *
+     * @param mixed $value The value to check
+     * @return bool
+     */
+    public static function isScalar($value)
+    {
+        return is_scalar($value);
```

Comment:
Please add a test for NULL, afaik NULL needs special handling as it is not "scalar" according to this check, but in our scope it should be OK to pass IMO.